pool/hub.go

```diff
@@ -716,7 +716,7 @@ func (h *Hub) FetchWorkQuotas() ([]*Quota, error) {
 func (h *Hub) AccountExists(accountID string) bool {
 	_, err := h.cfg.DB.fetchAccount(accountID)
 	if err != nil {
-		log.Tracef("Unable to fetch account for id: %s", accountID)
+		log.Errorf("unable to fetch account for id: %s", accountID)
```

Comment:
I dont agree with this being an error. This is used by the UI to check if an account ID provided by a site visitor exists - there is not an error or problem if the user provides an invalid ID - they could be on the wrong pool, or made a typo. Trace or Info level maximum imo.

Reply:
Noted, reverting.